x-pack/legacy/plugins/security/index.js

```diff
@@ -127,7 +126,7 @@ export const security = (kibana) => new kibana.Plugin({
       isSystemAPIRequest: server.plugins.kibana.systemApi.isSystemApiRequest.bind(
         server.plugins.kibana.systemApi
       ),
-      cspRules: createCSPRuleString(config.get('csp.rules')),
+      cspRules: config.get('csp.header'),
```

Comment:
Couldn't these be accessed via `server.newPlatform.setup.core.http.csp.header`?

Reply:
Ugh, yes!